packages/minimongo/minimongo.js

```diff
@@ -180,6 +180,42 @@ LocalCollection.Cursor.prototype.forEach = function (callback, thisArg) {
   });
 };
 
+LocalCollection.Cursor.prototype[Symbol.iterator] = function () {
+  var self = this;
```

Comment:
I just wanted to keep it similar to the rest of the code. Especially because code is very similar to `forEach`. I do not really care though. :-)